crates/context_aware_config/src/api/custom_types/handlers.rs

```diff
@@ -0,0 +1,160 @@
+use crate::db::models::JsonSchemaTypes;
+use crate::db::schema::jsonschema_types::{self, dsl};
+use actix_web::web::{Json, Path, Query};
+use actix_web::{delete, get, post, put, HttpResponse, Scope};
+use chrono::Utc;
+
+use diesel::{ExpressionMethods, QueryDsl, RunQueryDsl};
+
+use jsonschema::JSONSchema;
+use regex::Regex;
+use serde_json::{json, Value};
+use service_utils::{
+    bad_argument, db_error, result as superposition, service::types::DbConnection,
+    unexpected_error,
+};
+use superposition_types::User;
+
+use crate::api::custom_types::types::{QueryFilters, TypeTemplateRequest};
+
+const TYPE_NAME_REGEX: &str = "^[a-zA-Z0-9-_]{1,64}$";
+
+pub fn endpoints() -> Scope {
+    Scope::new("")
+        .service(list_types)
+        .service(create_type)
+        .service(update_type)
+        .service(delete_type)
+}
+
+#[post("")]
+async fn create_type(
+    request: Json<TypeTemplateRequest>,
+    db_conn: DbConnection,
+    user: User,
+) -> superposition::Result<HttpResponse> {
+    let DbConnection(mut conn) = db_conn;
+    let _ = JSONSchema::compile(&request.type_schema).map_err(|err| {
```

Comment:
```suggestion
   JSONSchema::compile(&request.type_schema).map_err(|err| {
```

Reply:
This is the avoid a warning with the compiler telling us that we are not using the result returned.
https://users.rust-lang.org/t/what-is-the-best-way-to-ignore-a-result/55187